### PR TITLE
Alarm support

### PR DESCRIPTION
**Pull request by thepwagner**
- Every alarm must be explicitly defined.
- Alarms may trigger PagerDuty, Slack, Email; or any combinations thereof!
- ASG metrics are supported for parity
- Every ELB metric is available, 🎊 !

### CODE REVIEW

File: src/lambda/sns-to-slack.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
+/**
+ * @author DeviaVir
```
Comment (DeviaVir) [inline src/lambda/sns-to-slack.js]: this hack lives on! One day it will deserve its own repo, I swear

**Comment by DeviaVir** [comment]
didn't this have CI testing with jenkins?

**Comment by thepwagner** [comment]
@DeviaVir 
Jenkins was building, but didn't have permission to publish results (fixed).
Jenkins build was broken by https://github.com/pebble/spacel-provision/pull/2 (fixed).

**Comment by lrvick** [comment]
Well defined abstractions and tests and overall pretty easy to read. +1

**Pull request merged**